src/main/java/xyz/nucleoid/bedwars/game/active/shop/BwTeamShop.java

```diff
@@ -114,4 +114,7 @@ public static ShopUi create(ServerPlayerEntity player, BwActive game) {
     private static int stagedUpgrade(int first, int level) {
         return MathHelper.floor(Math.pow(2, level) * first);
     }
+    private static int totalCost(BwParticipant participant, BwActive game, float original) {
+        return (int) game.participantsFor(participant.team).count() <= 2 ? (int) original : (int) (original * 2);
```

Comment:
Minor thing- but this could rather use the player set on `TeamState` to avoid the stream. Also, maybe call this method something like `teamScaledCost`?